javascript/exercises/cashRegisterSolution.js

```diff
@@ -0,0 +1,50 @@
+var carts = {
+  brenda: { 
+    banana: "1.25",
+    handkerchief: ".99",
+    Tshirt: "25.01"
+    },
+  claire: {
+    "apple": "0.60",
+    "nalgene": "10.34",
+    "protein_shake": "22.36"
```

Comment:
I'm glad you took the `$` out, because you cover it in the other string exercise 👍